src/cmd/cli/command/mcp.go

```diff
@@ -51,12 +51,12 @@ var mcpServerCmd = &cobra.Command{
 		// Create a new MCP server
 		term.Debug("Creating MCP server")
 		s := server.NewMCPServer(
-			"Defang Services",
+			"Defang MCP Server",
 			RootCmd.Version,
 			server.WithResourceCapabilities(true, true), // Enable resource management and notifications
 			server.WithPromptCapabilities(true),         // Enable interactive prompts
 			server.WithToolCapabilities(true),           // Enable dynamic tool list updates
-			server.WithInstructions("You are an MCP server for Defang Services. Your role is to manage and deploy services efficiently using the provided tools and resources."),
+			server.WithInstructions("You are an MCP server for Defang and do not use the Defang CLI. Your role is to manage and deploy services efficiently using the provided tools and resources. When interacting with the LLM make it use the MCP tools to perform actions and do not use the Defang CLI directly. When ask about defang please use the MCP resources to answer."),
```

Comment:
Let's keep this simple for the moment and direct it at the language model using the tool.
```suggestion
			server.WithInstructions("Use these tools to deploy projects to the cloud with Defang. These tools also help manage deployed projects, manage config variables, and estimate the monthly cost of deploying a given compose file."),
```